tardis/montecarlo/base.py

```diff
@@ -340,7 +340,7 @@ def run(
         self._initialize_geometry_arrays(model)
 
         self._initialize_packets(
```

Comment:
This function doesn't have a docstring. Would you mind adding that?

Reply:
Added!